controls/cis_ocp_1_4_0/section-1.yml

```diff
@@ -322,17 +322,19 @@ controls:
       levels: level_1
   - id: '1.4'
     title: Scheduler
-    status: pending
+    status: automated
     rules: []
     controls:
     - id: 1.4.1
       title: Ensure that the healthz endpoints for the scheduler are protected by RBAC
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - rbac_debug_role_protects_pprof
       levels: level_1
     - id: 1.4.2
       title: Verify that the scheduler API service is protected by RBAC
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - rbac_debug_role_protects_pprof
```

Comment:
> @rhmdnd I'm having a hard time connecting what the rule is checking for (`/debug/pprof`), to the rule text.
> 
> I found this rule with similar text and reference that is checking for `/metrics`. https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/api-server/api_server_profiling_protected_by_rbac/rule.yml
> 
> Is this rule not appropriate? Should its references be removed?
> 

Good catch - I pulled that into it's own change since it's updating section 1.2.

https://github.com/ComplianceAsCode/content/pull/10848

I guess the question becomes do we want to reuse the API server rule here for the scheduler.

> --
> 
> I also found the following rules which have references seem to be aligned with CIS v1.1:
> 
>     * https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/scheduler/scheduler_no_bind_address/rule.yml
> 
>     * https://github.com/ComplianceAsCode/content/blob/master/applications/openshift/scheduler/scheduler_port_is_zero/rule.yml
> 
> 
> What do you think of removing their references?

We can remove them - I don't think they're used in the latest benchmark.